src/components/explorer/IndividualExperiments.js

```diff
@@ -352,8 +361,27 @@ const IndividualExperiments = ({ individual }) => {
 };
 
 // expand here accordingly
-const isDownloadable = (result) =>
-    ["vcf", "cram", "bw", "bigwig"].includes(result.file_format?.toLowerCase());
+const isDownloadable = (result) => {
```

Comment:
just remove the isDownloadable function/check - if the file exists it should be downloadable